docs/using_eessi/building_on_eessi.md

```diff
@@ -6,67 +6,113 @@ Building on top of EESSI with EasyBuild is relatively straightforward. One cruci
 ### Starting the EESSI software environment
 Start your environment as described [here](../using_eessi/setting_up_environment.md)
 
-### Configure EasyBuild
-To configure EasyBuild, first, check out the [EESSI software-layer repository](https://github.com/EESSI/software-layer.git). We advise you to check out the branch corresponding to the version of EESSI you would like to use.
+### Using the EESSI-extend module
+The `EESSI-extend` module facilitates building on top of EESSI using EasyBuild. It does a few key things:
 
-If you are unsure which version you are using, you can run
+1. It configures EasyBuild to match how the rest of the EESSI software is build
+2. It configures EasyBuild to use a certain installation path (e.g. in your homedir), taking into account the hardware architecture you are building on
+3. It adds the relevant subdirectory from your installation path to your `MODULEPATH`, to make sure your newly installed modules are available
+4. It loads the EasyBuild module
+
+The `EESSI-extend` module recognizes a few environment variables. To print an up-to-date list, check the module itself
 ```
-echo ${EESSI_VERSION}
+module help EESSI-extend/2023.06-easybuild
 ```
-to check it.
 
-To build on top of e.g. version `2023.06` of the EESSI software stack, we check it out, and go into that directory:
+The key environment variables it will respect are:
+ - EESSI_USER_INSTALL can be set to a location to install modules for use by the user only. The location must already exist on the filesystem. Set this environment variable if you're an end-user who wants to install additional software just for yourself.
+ - EESSI_PROJECT_INSTALL can be set to a location to install modules for use by a project. The location must already exist on the filesystem and you should ensure that the location has the correct Linux group and the SGID permission is set on that directory (`chmod g+s $EESSI_PROJECT_INSTALL`) so that all members of the group have permission to read and write installations. Set this environment variable if you're an end-user who's part of a project, and wants to create additional installations that should be available to all project members.
```

Comment:
```suggestion
 - `EESSI_PROJECT_INSTALL` can be set to a location to install modules for use by a project. The location must already exist on the filesystem and you should ensure that the location has the correct Linux group and the SGID permission is set on that directory (`chmod g+s $EESSI_PROJECT_INSTALL`) so that all members of the group have permission to read and write installations. Set this environment variable if you're an end-user who's part of a project, and wants to create additional installations that should be available to all project members.
```